lib/charliecloud.py

```diff
@@ -258,6 +292,17 @@ def download(self, use_cache):
             dl.get_layer(lh, path)
       dl.close()
 
+   def download_fat_manifest(self, dl=None, use_cache=True):
```

Comment:
Seems awkward that the fat manifest gets its own download method but the other components don't.

Reply:
Two different functions use it: `print_arch_list` and `set_digest_for_arch`. In both functions there is a use case in which the fat manifest for a specified image hasn't been downloaded yet.